license_key.go

```diff
@@ -15,7 +15,7 @@ type LicenseKeyAttributes struct {
 	KeyShort        string      `json:"key_short"`
 	ActivationLimit int         `json:"activation_limit"`
 	InstancesCount  int         `json:"instances_count"`
-	Disabled        int         `json:"disabled"`
+	Disabled        bool        `json:"disabled"`
```

Comment:
Fix the `license_key_service_test`

Reply:
Thank you for the reminder, the tests have been fixed. I'll be more careful next time 😂